From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: genetlink: remove genl_bind
Patch-mainline: never, kabi
References: kabi

In networking-stable-20_07_17, commit 1e82a62fec61 (genetlink: remove
genl_bind) removed 2 members from struct genl_family. This made the kABI
checker complaining.

Reintroduce the members as placeholders.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/net/genetlink.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/include/net/genetlink.h
+++ b/include/net/genetlink.h
@@ -55,6 +55,9 @@ struct genl_family {
 	void			(*post_doit)(const struct genl_ops *ops,
 					     struct sk_buff *skb,
 					     struct genl_info *info);
+	/* these 2 are kABI placeholders */
+	int			(*mcast_bind)(struct net *net, int group);
+	void			(*mcast_unbind)(struct net *net, int group);
 	struct nlattr **	attrbuf;	/* private */
 	const struct genl_ops *	ops;
 	const struct genl_multicast_group *mcgrps;
